assets/src/styles/blocks/Covers/CampaignCovers.scss

```diff
@@ -105,7 +105,8 @@
         text-align: center;
         font-size: $font-size-md;
         font-family: $roboto;
-        color: $yellow;
+        background-color: var(--campaign-covers-block_yellow-cta_background_color, transparent);
```

Comment:
This should probably be renamed? `yellow-cta` is a bit too "color specific", and it really doesn't make sense if NROs are going to change it... Also it's overriden very often in campaign themes, sometimes directly like [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/styles/campaigns/themes/_theme_forest.scss#L159)) or using mixins

Reply:
Yeah especially for campaigns there are many complicated cases like this one. Maybe for now it's easier if we leave them out and start with the more clear-cut ones.

Though ultimately it hints at the issue we're addressing with the css variables. If we condense it to what we really need it could be
```
.cta {
  // There is actually only the yellow-cta class on this block currently, so we probably better drop the color prefix there.
  background-color: var(--covers-block-cta-background-color, $yellow);
}
```
where campaigns would then just have the right value specified in their variables.

Reply:
Something we can consider here too is that you can fallback to another variable. That would allow more granular control if needed but still apply the general cta color variable otherwise.
```
  background-color: var(--cta-background-color, var(--covers-block-cta-background-color, $yellow));
```

Reply:
Maybe we can leave this one out for now and create a separate ticket to take care of it in the future? (renaming `yellow-cta` to something more generic and find a way make it more customisable for child-themes)